src/components/uielements/assets/assetCard/AssetCard.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useCallback, useRef, useState, useEffect } from 'react'
+
+import { TokenAmount, tokenAmount } from '@thorchain/asgardex-token'
+import { bn, delay, formatBN } from '@thorchain/asgardex-util'
+import { Dropdown } from 'antd'
+import BigNumber from 'bignumber.js'
+import { sortBy as _sortBy } from 'lodash'
+
+import { clickedOutsideNode } from '../../../../helpers/eventHelper'
+import { PriceDataIndex } from '../../../../services/midgard/types'
+import { AssetPair } from '../../../../types/asgardex.d'
+import CoinInputAdvanced from '../../coins/coinInputAdvanced'
+import Label from '../../label'
+import Selection from '../../selection'
+import {
+  AssetCardFooter,
+  AssetData,
+  AssetNameLabel,
+  CardBorderWrapper,
+  CardTopRow,
+  AssetCardWrapper,
+  AssetDropdownButton,
+  AssetDropdownAsset,
+  AssetDropdownVerticalColumn,
+  DropdownIcon,
+  DropdownIconHolder,
+  FooterLabel,
+  HorizontalDivider
+} from './AssetCard.style'
+import AssetCardMenu from './AssetCardMenu'
+
+type DropdownCarretProps = {
+  open: boolean
+  onClick?: () => void
+  className?: string
+}
+const DropdownCarret: React.FC<DropdownCarretProps> = ({ open, onClick = () => {}, className = '' }): JSX.Element => {
+  const onClickHandler = useCallback(() => {
+    onClick()
+  }, [onClick])
+
+  return (
+    <DropdownIconHolder>
+      <DropdownIcon open={open} className={className} type="caret-down" onClick={onClickHandler} />
+    </DropdownIconHolder>
+  )
+}
+
+type Props = {
+  asset: string
+  assetData?: AssetPair[]
+  amount: TokenAmount
+  price: BigNumber
+  priceIndex?: PriceDataIndex
+  unit?: string
+  slip?: number
+  title: string
+  searchDisable?: string[]
+  withSelection?: boolean
+  withSearch?: boolean
+  onSelect?: (value: number) => void
+  onChange?: (value: BigNumber) => void
+  onChangeAsset?: (asset: string) => void
+  className?: string
+  dataTestWrapper?: string
+  dataTestInput?: string
+  children?: React.ReactNode
+  inputProps?: {
+    'data-test': string
+  }
+  'data-test'?: string
```

Comment:
Pls remove all `data-test` in any component. These legacy code coming from BEPSwap, but we don't need this in ASGARDEX.